src/Aspire.Hosting.Redis/PublicAPI.Shipped.txt

```diff
@@ -7,7 +7,6 @@ Aspire.Hosting.ApplicationModel.RedisResource.RedisResource(string! name) -> voi
 Aspire.Hosting.Redis.RedisCommanderResource
 Aspire.Hosting.Redis.RedisCommanderResource.RedisCommanderResource(string! name) -> void
 Aspire.Hosting.RedisBuilderExtensions
-static Aspire.Hosting.RedisBuilderExtensions.AddRedis(this Aspire.Hosting.IDistributedApplicationBuilder! builder, string! name, int? port = null) -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.ApplicationModel.RedisResource!>!
```

Comment:
We shouldn't be editing this file. But instead you can add a `*REMOVED*` line to the Unshipped.txt file. See https://github.com/dotnet/aspire/pull/4327/files#diff-6a097377cfc55516be6e941106e3e5575efaa3ab602e773d13d76b803c68c989R25 for an example.